### PR TITLE
EvalRegression - Add a flag to skip generating the R plot. Improve me…

### PR DESCRIPTION
**Pull request by JStehouwer**
…mory usage for very large input files.

### CODE REVIEW

File: openbr/core/eval.cpp [hunk 1]
```diff
@@ -1119,66 +1119,173 @@ class CorrelationCounter
...
+            values.append(QString("%1%2").arg(QString::number(preds[i])).arg(separator));
+        values.append(QString("%1").arg(preds[preds.size()-1]));
+        return values;
+    }
+
+    void print_hist()
+    {
+        int num_samples = gts.size();
+        int histSize = 20;
+        unsigned int* hist = new unsigned int[histSize*histSize];
```
Comment (Keyurpatel93) [inline openbr/core/eval.cpp:1168]: use QVector

File: openbr/core/eval.cpp [hunk 2]
```diff
@@ -1119,66 +1119,173 @@ class CorrelationCounter
...
+        }
+        printf(" 100.0  |\n\n");
+
+        delete [] hist;
+    }
+};
+
+void EvalRegression(const TemplateList predicted, const TemplateList truth, QString predictedProperty, QString truthProperty, bool generatePlot) {
+    float rms = 0.f, mae = 0.f;
+    CorrelationCounter *cc = new CorrelationCounter();
```
Comment (Keyurpatel93) [inline openbr/core/eval.cpp:1226]: Delete allocated memory

File: openbr/core/eval.cpp (hunk 1 above)
Comment (JStehouwer) [reply openbr/core/eval.cpp]: Resolved in cf4912b.

File: openbr/core/eval.cpp (hunk 2 above)
Comment (JStehouwer) [reply openbr/core/eval.cpp]: Resolved in cf4912b.

**Pull request merged**